lib/autobuild/packages/cmake.rb

```diff
@@ -283,6 +291,7 @@ def common_utility_handling( # rubocop:disable Metrics/ParameterLists
                     if internal_doxygen_mode?
                         run_doxygen
                     else
+                        args = "ARGS=#{test_args.map { |a| %("#{a}") }.join(' ')}"
```

Comment:
common_utility_handling is also used for docs. Keep the *args argument, and pass the test args from the `with_tests` handler (where you removed the hardcoded ARGS=-V)

Reply:
That was actually my first idea but won't work well because the test Rake task is created too early when the package is defined and can't be replaced so the user won't be able to override. The arguments really have to be evaluated in the task block.

Reply:
Ah, wait, I think passing the array object rather than the evaluated string (as I've done before) should work.